app/services/exports/export_distributions_csv_service.rb

```diff
@@ -63,8 +65,11 @@ def base_table
         "Source Inventory" => ->(distribution) {
           distribution.storage_location.name
         },
-        "Total Items" => ->(distribution) {
-          distribution.line_items.total
+        "Total number of Items" => ->(distribution) {
+          # filter the line items by item id (for selected item filter) to
+          # get the number of items
+          distribution.line_items.find_by(item_id: @filtered_item_id)&.quantity ||
+            distribution.line_items.total
```

Comment:
What happens if the `item_id` isn't found? Is quantity 0 or nil?

Separate question: Does this end up doing two separate DB queries if we're not filtering? If so, might make sense to do the (admittedly less succinct) Ruby if/then logic to determine what to return rather than counting on the first query to return `nil`.

Reply:
when an item_id is not found, yes it would return nil. Yes makes sense to query the DB once based on whether the filtered item is present or not.